packages/pretty-format/src/__tests__/pretty_format.test.js

```diff
@@ -549,6 +549,63 @@ describe('prettyFormat()', () => {
     }).toThrow();
   });
 
+  it('throws PrettyFormatPluginError if test throws an error', () => {
+    const options = {
+      plugins: [
+        {
+          print: () => '',
+          test() {
+            throw new Error('Where is the error?');
+          },
+        },
+      ],
+    };
+
+    try {
+      prettyFormat('', options);
+    } catch (error) {
+      expect(error.name).toBe('PrettyFormatPluginError');
+    }
```

Comment:
If for any reason the `prettyFormat` call doesn’t throw, then the test passes?

What do you think about:

```js
let name;
try {
  prettyFormat('', options);
} catch (error) {
  name = error.name;
}
expect(name).toBe('PrettyFormatPluginError');
```

Reply:
Either do `expect().toThrow()` or `expect.hasAssertions` instead

Reply:
@SimenB Thank you! Because `toThrow` matches the `message` but the `name` is the test criterion, then our requested change is keep try-catch as originally written and insert `expect.hasAssertions()` at beginning of each of the 3 test blocks?

Reply:
True. Or `.toThrow(PrettyFormatPluginError)` if it's exported.